src/widgets/dashboardOverview.js

```diff
@@ -28,13 +28,40 @@ export default async function dashboardOverview(scopes) {
   const inPerson = duration.filter((report) => report.deliveryMethod.toLowerCase() === 'in-person').length
     .toString();
 
+  // Get Region.
+  const region = query['region.in'];
+
+  // Get Start and End Date.
+  const dateRange = query['startDate.win'];
+  if (!dateRange) {
+    return null;
+  }
+  const dates = dateRange.split('-');
+
+  let startDte;
+  let endDte;
+
+  // Check if we have a Start Date.
+  if (dates.length > 0) {
+    // eslint-disable-next-line prefer-destructuring
+    startDte = dates[0];
+  }
+
+  // Check if we have and End Date.
+  if (dates.length > 1) {
+    // eslint-disable-next-line prefer-destructuring
+    endDte = dates[1];
+  }
+
+  const participantsWhere = `WHERE "regionId" in (${region !== undefined ? region : '0'})  AND "status" = '${REPORT_STATUSES.APPROVED}' AND "startDate" >= '${startDte}' AND "startDate" <= '${endDte}'`;
+
   const res = await ActivityReport.findAll({
     attributes: [
       // This is a literal because it needs to *not* respect the scopes passed in
       // [sequelize.literal(`(SELECT COUNT(*) from "Grants" ${grantsWhere})`), 'numTotalGrants'],
       [sequelize.fn('COUNT', sequelize.fn('DISTINCT', sequelize.col('"ActivityReport".id'))), 'numReports'],
       [sequelize.fn('COUNT', sequelize.fn('DISTINCT', sequelize.col('"activityRecipients->grant"."id"'))), 'numGrants'],
-      [sequelize.fn('COUNT', sequelize.fn('DISTINCT', sequelize.col('"activityRecipients->nonGrantee"."id"'))), 'nonGrantees'],
+      [sequelize.literal(`(SELECT COALESCE(SUM("numberOfParticipants"), 0) FROM "ActivityReports" ${participantsWhere})`), 'numParticipants'],
```

Comment:
Isn't this making it's own call to the `ActivityReports` (as a subquery)? If so it isn't respecting the scopes below in the `where` property. I would think we can do the select on `numberOfParticipants` much in the same was as `numReports` or `numGrants` above.

Reply:
Yes this is making a call to activity reports. However because it's joined to the activity recipients table it will cause double counting (Cartesian Product). For example, if one activity reports has two grantees it will double count.

Reply:
What should this column be counting? number of grantees, grants, or participants? There is a field in the "ActivityReports" table called `numberOfParticipants` that this is currently counting. That field is different then the number of grants or grantees in the report. Users enter a number for the field (so I guess they could be entering in the number of grants/grantees but it definitely isn't guaranteed). If we want to know the number of grants/non-grantees we would need to sum up all distinct "ActivityReportRecipients"

Reply:
After looking at this a bit more I think a possible solution is to move the `numberOfParticipants` out into it's own SQL call. The new call can inner join on the "ActivityRecipients" and "Grants" tables, which will allow us to properly do a sum on `numberOfParticipants`. The new SQL call could then use the scopes without having to worry about buliding it's own different "where" clause.